docs/src/code_of_conduct.md

```diff
@@ -0,0 +1,131 @@
+# [Contributor Covenant Code of Conduct](@id code-of-conduct)
```

Comment:
Please remove this redundant copy and use a setup such as 

https://github.com/ranocha/SummationByPartsOperators.jl/blob/0206a74140d5c6eb9921ca5021cb7bf2da1a306d/docs/make.jl#L27-L41

to create the file automatically from the docs.

Reply:
Fixed in 9f9c612f.